build/pipelines/templates/build-console-public.yaml

```diff
@@ -19,7 +19,19 @@ jobs:
     inputs:
       packageType: sdk
       version: 6.0.x
-      installationPath: $(Agent.ToolsDirectory)/dotnet
+
+  - task: UseDotNet@2
+    displayName: 'Use .NET 7 SDK'
+    inputs:
+      packageType: sdk
+      version: 7.0.x
+
+  - task: UseDotNet@2
+    displayName: 'Use .NET 8 SDK'
+    inputs:
+      packageType: sdk
+      version: 8.0.x
+      includePreviewVersions: true
```

Comment:
this does not need to be a preview version anymore

Reply:
True, corrected.